src/components/ProfileMenu/ProfileMenu.component.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import './ProfileMenu.styles.css';
+
+const MenuSeparator = styled.div`
+  border: 1px solid #ffffff;
+  margin: 20px;
+  width: 90%;
+`;
+
+const MenuItem = styled.div`
+  margin: 20px;
+`;
+
+function ProfileMenu(props) {
+  const { className } = props;
+  return (
+    <div className={`profile-menu-container ${className}`}>
```

Comment:
I suggest to use css modules like: `import styles from './ProfileMenu.module.css'`, and then use it like:
`<div className={styles['profile-menu-container'] ${className}`}>`

On the other hand, you should better use another prop to check if the profile is either active or inactive:
```
...
const { active } = props;
return (
     <div className={clsx(styles['profile-menu-container'], active? styles['active']: styles['inactive'] )}
...
```
That way you avoid mixing classes of this component in places where they shouldn't be